scripts/genunicodetables.lua

```diff
@@ -0,0 +1,227 @@
+#!/usr/bin/lua
```

Comment:
What for? Other generator scripts are not there, VimL is no longer used. And former `runtime/tools/unicode.vim` was creating a number of buffers in whatever directory it was run (it was needed to buffer contents to `mbyte.c` manually then) and new script is not going to do anything like this (it does not run Vim in first place).